service/src/main/java/gov/nasa/pds/api/registry/model/identifiers/LidVidUtils.java

```diff
@@ -120,33 +109,36 @@ public static List<String> getAllLidVidsByLids(
     }
 
     public static String resolve (
-    		String identifier,
+    		String _identifier,
```

Comment:
Variable names should not start with underscore _ or dollar sign $ characters, even though both are allowed.
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=Variable%20names%20should%20not%20start,should%20be%20short%20yet%20meaningful.
https://google.github.io/styleguide/javaguide.html

Reply:
Roger that, will fix - this is an artifact of me not having yet converted the function's interface to use the classes.

Reply:
resolved in 9c6653c19c98f0b268887f11f4e8dc2f6736b268